create_issues.rb

```diff
@@ -0,0 +1,58 @@
+module CreateIssues
+
+  def create_issues
```

Comment:
Please group actions into a several separate methods since we can see here a lot of lines of code and it's not clear enough to reader